MVC/view/macGUI.py

```diff
@@ -27,14 +31,10 @@ def __init__(self, parent, ctrl):
         self.parent.config(menu=self.menu)
         self.img.place(x = 0,y = 0)
         # created an input box
-        self.inputFrame = tk.Frame(self.parent)
```

Comment:
Why was this deleted? Please add it back!

Reply:
Unless you deleted it for a reason but please specify if so!

Reply:
also deleted more than just this line check lines 30-35, and 37